README.md

```diff
@@ -204,11 +204,44 @@ These build targets are supported:
 | electron      | Builds the Electron binary **(Default)**                 |
 | electron:dist | Builds the Electron binary and generates a dist zip file |
 | mksnapshot    | Builds the `mksnapshot` binary                           |
-| node:headers  | Builds the node headers `.tar.gz` file                   |
+| electron:node_headers  | Builds the node headers `.tar.gz` file          |
+| electron:electron_symbols  | Generate the breakpad symbols in release builds     |
```

Comment:
I think this might cause some confusion.

Unfortunately the semantics between `e build [target]` and `e build --target [target]` are kind of confusing. They're both called `[target]` but in the former it's really just some aliases we define to ninja targets, and in the latter it's ninja targets.

This leads to weirdness like `e build mksnapshot` is building `electron:electron_mksnapshot_zip` while `e build --target mksnapshot` is actually building `v8:mksnapshot`.

Currently the table only lists the aliases, which aligns with the sentence that proceeds it about using `e build [target]`. These changes here introduce ninja targets into the table, which *kind of* work with `e build [target]` but arguably shouldn't. If you run `e build v8:mksnapshot` the output indicates that `autoninja v8:mksnapshot -j 200 electron` is being run, which seems like unexpected behavior and a bug.

Reply:
@dsanders11 thinking about it - it's ultimately probably easier to do away with the aliases since the average use won't need them and it's better to be explicit in e.g. ci 🤔 What do you think?

Reply:
I like that idea; I'm fine with removing the aliases

Reply:
I like that idea; I'm fine with removing the aliases.

@dsanders11 ping